src/wallet/rpcwallet.cpp

```diff
@@ -2993,7 +2993,7 @@ void FundTransaction(CWallet* const pwallet, CMutableTransaction& tx, CAmount& f
             if (options.exists("changeAddress")) {
                 throw JSONRPCError(RPC_INVALID_PARAMETER, "Cannot specify both changeAddress and address_type options");
             }
-            coinControl.m_change_type = pwallet->m_default_change_type;
+            coinControl.m_change_type.emplace();
```

Comment:
fa078530bc3824d7d5a1cc4f39b617054b08d77e

nit, same thing, use temporary.

Reply:
changed for same reason